src/legacy/ui/public/vislib/visualizations/point_series/_point_series.js

```diff
@@ -19,13 +19,22 @@
 
 import _ from 'lodash';
 
+const thresholdLineDefaults = {
+  show: false,
+  value: 10,
+  width: 1,
+  style: 'full',
+  color: '#00B3A4'
```

Comment:
@FilipRy One missed placed to use the EUI palette service. Also Caroline left another comment suggesting we use the `[9]` not the `[0]` as a default color of that palette. Could you perhaps use the palette service here and change all the places to use `[9]` by default? Thanks. After that the PR looks ready for a last review and merge to me.

Reply:
@timroes I guess the PR is ready for the last review at the moment.